src/libcollections/borrow.rs

```diff
@@ -274,3 +274,13 @@ impl<'a, T: ?Sized + ToOwned> AsRef<T> for Cow<'a, T> {
         self
     }
 }
+
+// // Generic implementation mentioned in
```

Comment:
I had included it because of @rphmeier's earlier comment, but I can remove it again.